### ISSUE TITLE
update package for sublime 3.1.1

### ISSUE THREAD

**Issue by WISTFUL12** [doobleweb/RTL-Mirror#3]
In sublime 3.0, Build 3143 this package works fine but after upgrade to sublime 3.1.1 build 3176 the issue comes back again!
I have changed "font_options" to ["gdi"]

**Comment by sahar-manavate** [comment]
yes, there was a change in the text rendering starting with that version.
the fix you mentioned is the only way to fix it so far.
will look into making it automated.

**Comment by WISTFUL12** [comment]
Changing "font_options" to ["gdi"] fix text rendering but doe's not fix text direction (in sublime 3.1.1)

**Comment by sarirgit** [comment]
@sahar-manavate @SaharZehavi 
When you release new version of your efficient plugin? (with fix this issue)

**Comment by kultuk** [comment]
at the moment i don't know when i'll get into it. hopefully soon but can't make any promises.
feel free to fix it and PR the repository.

**Comment by SaharZehavi** [comment]
i inserted a small fix that should cover that problem. you are more then welcomed to try it out.

**Comment by sarirgit** [comment]
Thanks...
Now characters are displayed as separate

![image](https://user-images.githubusercontent.com/40927999/42549432-62041f84-84e1-11e8-806b-eedb3d2713d1.png)

**Comment by mr-islam** [comment]
Same issue with the latest sublime text release
